portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,25 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> names;
+
+  @Override
+  public void init(){
+    names = new ArrayList<>();
+    names.add("Alex");
+    names.add("Ana");
+    names.add("Ion");
+    names.add("Paul");
+    names.add("Robert");
+    names.add("Andrei");
+    names.add("Matei");
+    names.add("Radu");
+  }  
+
```

Comment:
Hmm, this was present in the previous PR. You should branch from the latest branch if you've pending changes, not from main.

Reply:
Solved